static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -19,63 +20,126 @@ type Props = {
 
 function DomMutations({replay}: Props) {
   const {isLoading, actions} = useExtractedCrumbHtml({replay});
+  const {
+    setCurrentTime,
+    highlight,
+    setCurrentHoverTime,
+    clearAllHighlights,
+    removeHighlight,
+  } = useReplayContext();
 
   const startTimestampMs = replay.getReplay().started_at.getTime();
 
+  const handleMouseEnter = useCallback(
+    (item: Crumb) => {
+      if (startTimestampMs) {
+        setCurrentHoverTime(relativeTimeInMs(item.timestamp ?? '', startTimestampMs));
+      }
+
+      if (item.data && 'nodeId' in item.data) {
+        // XXX: Kind of hacky, but mouseLeave does not fire if you move from a
+        // crumb to a tooltip
+        clearAllHighlights();
+        highlight({nodeId: item.data.nodeId, annotation: item.data.label});
+      }
```

Comment:
you don't need to copy+paste this into here from breadcrumbs:

a) it's copy+paste, shouldn't be doing that
b) none of the console breadcrumbs that we render in this list will have `item.data.nodeId` values.